README.md

```diff
@@ -59,4 +59,4 @@ These binary files are copied from Steamworks SDK's `redistribution_bin` directo
 
 ## Resources
 
- * [Steamworks SDK](https://partner.steamgames.com/doc/sdk)
+ * [Steamworks SDK](https://partner.steamgames.com/doc/sdk)
```

Comment:
What's the change?

Reply:
It didn't change, but these prompts appeared automatically after submission :see_no_evil:

Reply:
OK then revert the change.

Reply:
done